docs/includes/ksql-includes.rst

```diff
@@ -333,3 +333,16 @@ To enable JMX metrics, set ``JMX_PORT`` before starting the KSQL server:
 
     $ export JMX_PORT=1099 && \
       <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+.. log limitations
+
+.. important:: By default KSQL will attempt to store its logs in a directory called ``/logs`` that is relative to the location
```

Comment:
It should be `logs`, not `/logs`. 

Suggestion:

> By default KSQL will attempt to store its logs in a directory called ``logs`` that is relative to the location of the `ksql` executable. For example, if `ksql` is installed at `/usr/local/bin/ksql`, then it would attempt to store its logs under `/usr/local/logs`. If you are starting ...